demos/exit/exit.go

```diff
@@ -0,0 +1,97 @@
+package main
+
+import (
+	"image/color"
+	"log"
+	"math/rand"
+
+	"engo.io/ecs"
+	"engo.io/engo"
+)
+
+type GameWorld struct{}
+
+func (game *GameWorld) Preload() {
+
+	// This could be done individually: engo.Files.Add("data/icon.png"), etc
+	// Second value (false) says whether to check recursively or not
+	engo.Files.AddFromDir("data", false)
+
+	log.Println("Preloaded")
+}
+
+func (game *GameWorld) Setup(w *ecs.World) {
```

Comment:
When writing a demo, I want to *only* have the functionality which that demo is about - in order to avoid confusion.

Basically, would you mind removing all of the rendering code?